scripts/release/linear_update.sh

```
@@ -0,0 +1,49 @@
+fi
+
+# Read ticket IDs from the file and update each task
+while read -r ISSUE_ID; do
+    echo "Updating Linear issue $ISSUE_ID"
+
+    # Add a comment to the task
+    RESPONSE=$(curl -s -X POST "https://api.linear.app/graphql" \
+        -H "Authorization: $LINEAR_API_KEY" \
+        -H "Content-Type: application/json" \
+        --data '{
+          "query": "mutation { commentCreate(input: { issueId: \"'"$ISSUE_ID"'\", body: \"The task is part of the release v'$GITHUB_RELEASE_TAG' \" }) { success } }"
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)